independent-projects/arc/runtime/src/main/java/io/quarkus/arc/Arc.java

```diff
@@ -12,23 +12,37 @@ public final class Arc {
 
     private static final AtomicReference<ArcContainerImpl> INSTANCE = new AtomicReference<>();
 
+    /**
+     * Initializes {@link ArcContainer} with default settings.
+     * This is equal to using {@code Arc#initialize(ArcInitConfig.INSTANCE)}
+     *
+     * @return {@link ArcContainer} instance with default configuration
+     */
     public static ArcContainer initialize() {
```

Comment:
I kept the no-args variant without deprecation as I think it's better than always requiring config object and passing in the default instance.